doc/api/process.markdown

```diff
@@ -59,6 +59,9 @@ finished running the process will exit. Therefore you **must** only perform
 checks on the module's state (like for unit tests). The callback takes one
 argument, the code the process is exiting with.
 
+This event may not be fired if the process terminates due to signals like
```

Comment:
Fair enough. I should have been more clear when I provided the alternative wording, it wasn't cosmetic,  the exit event is emitted in only the conditions I mention: explicit exit, and implicit exit.